client/src/main/scripts/components/search/VideoResultsList.spec.tsx

```diff
@@ -17,13 +17,55 @@
  *
  * You can contact us at team@mytake.org
  */
+import React from "react";
 import * as renderer from "react-test-renderer";
 import { FoundationHarness } from "../../common/foundationTest";
-import { SearchMode, _searchImpl, _SearchWithData } from "./search";
+import {
+  SearchMode,
+  SearchResult,
+  _searchImpl,
+  _SearchWithData,
+} from "./search";
 // this data is generated by server/src/test/java/controllers/LucenePlayground.java
 import socialSecuritySearchResults from "./testData/socialSecuritySearchResults.json";
+import { VideoResultProps } from "./VideoResult";
 import { VideoResultsList } from "./VideoResultsList";
 
+jest.mock("./NGramViewer", () => ({
+  __esModule: true,
+  default: (props: SearchResult) => {
+    return <div>NGramViewer: {props.searchQuery}</div>;
+  },
+}));
```

Comment:
@nedtwigg fyi we had to do some special mocking here. By default, jest will try to write the entire data structure of the props object with a newline between each unique value in an array. In this case it was more than 5 million lines long.

Same below on line 56

Reply:
Thanks for tip!